### PR TITLE
Bugs in launcher.py

### PR DESCRIPTION
**Pull request by wendycwong**
discovered by Michal M

### CODE REVIEW

File: tracker/dmlc_tracker/launcher.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def main():
         (classpath, _) = subprocess.Popen('%s/bin/hadoop classpath' % hadoop_home,
                                           stdout=subprocess.PIPE, shell=True,
                                           env=os.environ).communicate()
-        for f in classpath.split(':'):
+        for f in classpath.decode('utf-8').split(':'):
```
Comment (szha) [inline tracker/dmlc_tracker/launcher.py]: assuming the change is for python2, could you verify it for python3 too?

Comment (wendycwong) [reply tracker/dmlc_tracker/launcher.py]: Nope, this is for python3.  If the changes was not made, the wrong path will be included.  I was trying to play with xgboost0.82 off hadoop/yarn.  I was having a hard time getting it to run.  Michal M discovered this bug and hence the proposed fix.

**Comment by hcho3** [comment]
@wendycwong Can you review #564? It addresses the same bug and supports both Python 2 and 3.

**Comment by hcho3** [comment]
Closing in favor of #564

**Pull request closed without merging**